hack/install-kwok.sh

```diff
@@ -0,0 +1,102 @@
+set -euo pipefail
+
+# get the latest version
+KWOK_REPO=kubernetes-sigs/kwok
+# KWOK_LATEST_RELEASE=$(curl "https://api.github.com/repos/${KWOK_REPO}/releases/latest" | jq -r '.tag_name')
+KWOK_LATEST_RELEASE="v0.3.0"
+
+# make a base directory for multi-base kustomization
+HOME_DIR=$(mktemp -d)
+BASE=${HOME_DIR}/base
+mkdir ${BASE}
+
+# allow it to tolerate everything, but not run on nodes that kwok launches
+cat <<EOF > "${BASE}/tolerate-all.yaml"
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kwok-controller
+  namespace: kube-system
+spec:
+  template:
+    spec:
+      tolerations:
+      - operator: "Exists"
```

Comment:
This is typically bad practice, since it means that this pod can never be evicted even if the node is experiencing issues. What was the reason we needed to tolerate everything?

Reply:
Good callout, was doing this just for my testing, I'll fix this.